src/config/database.config.ts

```diff
@@ -34,7 +34,8 @@ export class DatabaseConfig implements TypeOrmOptionsFactory {
         Question,
         QuestionOption,
       ],
-      synchronize: false,
+      synchronize: isDevelopment,
```

Comment:
Is this safe? This will try to update the tables as per entities. @vinayak-patil